src/qt/guiutil.cpp

```diff
@@ -399,7 +400,14 @@ bool openBitcoinConf()
     configFile.close();
 
     /* Open bitcoin.conf with the associated application */
-    return QDesktopServices::openUrl(QUrl::fromLocalFile(boostPathToQString(pathConfig)));
+    bool res = QDesktopServices::openUrl(QUrl::fromLocalFile(boostPathToQString(pathConfig)));
+#ifdef Q_OS_MAC
+    if (!res) {
```

Comment:
I think this needs a comment why this fallback is necessary.

Reply:
A comment like https://github.com/bitcoin/bitcoin/blob/master/src/qt/guiutil.cpp#L944 ?

Reply:
IMO, this issue is not a _Qt_ bug. This PR addressed macOS-specific behavior.
May I suggest a comment `// Workaround for macOS-specific behavior; see #15409.` ?

Reply:
Done